src/App.tsx

```diff
@@ -1,14 +1,48 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isLoader, setIsLoader] = useState(true);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+  const [selectValue, setSelectValue] = useState('all');
+  const [filterValue, setFilterValue] = useState('');
+
+  useEffect(() => {
+    const getTodosAsync = async () => {
+      const receivedTodos = await getTodos();
+
+      setTodos(receivedTodos);
+      setIsLoader(false);
+    };
+
+    getTodosAsync();
+  }, []);
+
+  const todosForRender
+    = todos
+      .filter(({ title, completed }) => {
+        switch (selectValue) {
+          case 'active':
+            return !completed && title.toLowerCase().includes(filterValue.toLowerCase());
```

Comment:
don't repeat code. Create a separate method that checks if filterValue includes in the title